tensorflow_addons/metrics/multilabel_confusion_matrix.py

```diff
@@ -24,6 +24,49 @@
 
 
 class MultiLabelConfusionMatrix(Metric):
+    """Computes Multi-label confusion matrix.
+
+    Class-wise confusion matrix is computed for the
+    evaluation of classification.
+
+    If multi-class input is provided, it will be treated
+    as multilabel data.
+
+    Consider classification problem with two classes.
+    Resultant matrix `M` will be in the shape of (num_classes, 2, 2).
```

Comment:
Mention that `num_classes=2`

Reply:
Added before `Resultant matrix `M` will be in the shape of ` line